src/lib/rest/rest.cpp

```diff
@@ -1410,7 +1410,22 @@ static int connectionTreat
   }
   else
   {
-    serveFunction(ciP);
+    if (metricsMgr.isOn())
+    {
+      struct timeval  start;
+      struct timeval  end;
+
+      gettimeofday(&start, NULL);
```

Comment:
Not sure if this is teh best strategy due to:

* The transaction has actually started much before (the start of `connectionTreat()` is around 200 lines above :)
* Before reaching this point, the transaction could have ended (due to error conditions). Note the `restReply()` cases abvoe this point.

Maybe the current implementation of `connectionTreat()` make this difficult. Maybe it is the moment to deal with https://github.com/telefonicaid/fiware-orion/issues/940 to easy things.

We can discuss this at skype on Monday.